app/Http/Controllers/HomeController.php

```diff
@@ -13,6 +18,13 @@ class HomeController extends Controller
      */
     public function index()
     {
-        return view('home');
+        $courses = Course::showCourseAtHomepage();
```

Comment:
@phamthimylinhhaui có thể tham khảo cách đặt tên anh comment ở bài của Toàn, Quân nhé